scripts/delete.sh

```diff
@@ -131,9 +131,15 @@ if [[ "$crb_ns" == "$namespace" ]]; then
   echo "deleting cluster scope RBAC"
   kustomize build rbac/cluster-scope | kubectl delete -f - --ignore-not-found=true
 fi
+
+# Determine whether deployment is namespace-scoped before deleting runtime resources
+is_namespace_scoped=$(kubectl exec deploy/modelmesh-controller -- printenv NAMESPACE_SCOPE)
 kustomize build default | kubectl delete -f - --ignore-not-found=true
 kustomize build rbac/namespace-scope | kubectl delete -f - --ignore-not-found=true
-kustomize build runtimes ${kustomize_load_restrictor_arg} | kubectl delete -f - --ignore-not-found=true
+if [[ ! $is_namespace_scoped ]]; then
```

Comment:
If MM was installed with your updated `install.sh` script:

```Bash
kubectl set env deploy/modelmesh-controller NAMESPACE_SCOPE=false
```

then I think this test will not work, since the variable `$is_namespace_scoped` does exist (is defined) but the string value is "false"

Reply:
Fixed